packages/utils/telemetry-utils/src/telemetryTypes.ts

```diff
@@ -78,6 +78,8 @@ export interface ITelemetryEventExt extends ITelemetryPropertiesExt {
 export interface ITelemetryGenericEventExt extends ITelemetryPropertiesExt {
 	eventName: string;
 	category?: TelemetryEventCategory;
+	sessionID?: string | undefined;
+	continuityID?: string | undefined;
```

Comment:
We cannot put these here, this is a very general interface for telemetry and we should not force every telemetry message that Fluid generates to have these properties :). But it shouldn't be necessary, we can already add arbitrary key/value pairs to the events.